espnet2/tasks/hubert.py

```diff
@@ -0,0 +1,458 @@
+import argparse
```

Comment:
I'm a little bit confusing.
Is this task for pretraining?

Reply:
Yes